Index: src/Whatsapp/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Whatsapp/index.js	(revision c023c60692d9feb3909f76e695b70770dc2c487b)
+++ src/Whatsapp/index.js	(date 1612874393221)
@@ -1,16 +1,17 @@
 import React, {useState, useEffect} from 'react';
 import {CountryList} from '../globalUtility/CONST'
 import {useHistory} from 'react-router-dom'
+import Header from "./header";
 import {Button, Modal, ModalHeader, ModalBody, ModalFooter, Col, Row, Card} from 'reactstrap';
-import {Popconfirm, Select, Avatar, Divider} from 'antd';
+import {Popconfirm, Select, Divider} from 'antd';
 import CreateIcon from '@material-ui/icons/Create';
 import DeleteIcon from '@material-ui/icons/Delete';
 import firebase from '../firebase'
 import "firebase/auth";
-import img1 from "../Assets/play store.png"
-import img2 from "../Assets/unnamed.png"
-import {createUserMessage, getUserAddress, createMessage, creatTag, getMessage, getTag, createBusinessAddress, deleteMessage, deleteTag, googleFirebase, deleteBussinessAddress, deleteHomeAddress, editMessage, editAddress, updateMess, updateAddre} from '../actions'
-import pic2 from '../Assets/images.jpg'
+import img1 from "../Assets/microSoft.png"
+import img2 from "../Assets/playStore.png"
+import img3 from "../Assets/appStore.png"
+import {createUserMessage, getUserAddress, createMessage, creatTag, getMessage, getTag, createBusinessAddress, deleteMessage, deleteTag, googleFirebase, deleteHomeAddress, editMessage, editAddress, updateMess, updateAddre} from '../actions'
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'antd/dist/antd.css';
 import '../App.css'
@@ -18,12 +19,39 @@
 import "./login.css";
 import {ArrowDownOutlined} from "@ant-design/icons";
 import {Bling as GPT} from "react-gpt";
+// import Header from "./header";
+// import $ from "jquery";
 
 const {Option} = Select;
 GPT.enableSingleRequest();
 
 const WhatsApp = () => {
 
+    // $("#onBold").click(function () {debugger
+    //     const selectedText = window.getSelection ? window.getSelection() : document.selection.createRange();  // second one for IE
+    //     console.log(selectedText.toString());
+    //     const s = selectedText.toString();
+    //     setTextBold(s);
+    //     // console.log(selectedText.toString())
+    //     // if (selectedText.getRangeAt) {
+    //     //     var range = selectedText.getRangeAt(0);
+    //     //     var newNode = document.createElement("span");
+    //     //     newNode.setAttribute('class', 'highlightedText');
+    //     //     range.surroundContents(newNode);
+    //     // } else {
+    //     //     selectedText.pasteHTML('<span class="highlightedText">' + selectedText.toString() + '</span>');
+    //     // }
+    //     $('textarea').highlight(selectedText.toString(),"highlight");
+    //     // $('.highlightedText').replaceWith(swapText);
+    //     //     $('#demoCustom').highlightTextarea({
+    //     //         words: [
+    //     //             { color : '#0F0',
+    //     //                 words  : ['World']
+    //     //             },
+    //     //         ]
+    //     //     });
+    // });
+
     const history = useHistory();
     const [user, setUser] = useState({});
     const [detail, setDetail] = useState({country: CountryList[0].code + CountryList[0].dial_code});
@@ -48,6 +76,7 @@
     const [AddressId, SetAddressId] = useState('');
     const [editMessageValue, setEditMessageValue] = useState({});
     const [editAddressValue, setEditAddressValue] = useState({});
+    // const [textBold, setTextBold] = useState("");
 
     const [modal, setModal] = useState(false);
     const toggle = () => setModal(!modal);
@@ -363,13 +392,15 @@
         } else {
             if (type === 'WhatsAppDesktop') {
                 const dialCode = "+".concat(detail.country.split("+")[1]);
-                const reDirectUrl = `https://api.whatsapp.com/send?phone=${dialCode}${detail.phone}&text=${detail.message}`;
+                const s = detail.message.replace(/\n/g, '%0a');
+                const reDirectUrl = `https://api.whatsapp.com/send?phone=${dialCode}${detail.phone}&text=${s}`;
                 window.open(reDirectUrl, '_blank');
                 resetForm();
                 setValidation({})
             } else if (type === 'WhatsAppWeb') {
                 const dialCode = "+".concat(detail.country.split("+")[1]);
-                const reDirectUrl = `https://web.whatsapp.com/send?phone=${dialCode}${detail.phone}&text=${detail.message}`;
+                const s = detail.message.replace(/\n/g, '%0a');
+                const reDirectUrl = `https://web.whatsapp.com/send?phone=${dialCode}${detail.phone}&text=${s}`;
                 window.open(reDirectUrl, '_blank');
                 resetForm();
                 setValidation({})
@@ -711,6 +742,7 @@
 
     return (
         <>
+            <Header user={user} text2={text2} confirm={confirmLogOut}/>
             <div className="whatsapp all646">
                 <Modal isOpen={modal} toggle={toggle}>
                     <ModalHeader toggle={toggle} style={{color: "black"}}/>
@@ -725,89 +757,20 @@
                             </div>
                         </div>
                     </ModalBody>
-
                 </Modal>
                 <Row>
                     <Col md={3}  style={{width: 320}}>
                         <div className="site-card-border-less-wrapper">
-                            {/*<Card title="Card title" >*/}
-                            {/*</Card>*/}
-                            <Card className="card" title="Card tittle" style={{width: "100%"}}>
-                                <div id="div-gpt-ad-1612775480075-0" >
-                                    <GPT adUnitPath='/419163168/https:////whatsdirectchat.com.Banner' slotSize={[[320, 50], [300, 100], [300, 250]]} />
-                                </div>
+                            <Card title="Card title" >
                             </Card>
+                            {/*<Card className="card" title="Card tittle" style={{width: "100%"}}>*/}
+                            {/*    <div id="div-gpt-ad-1612775480075-0" >*/}
+                            {/*        <GPT adUnitPath='/419163168/https:////whatsdirectchat.com.Banner' slotSize={[[320, 50], [300, 100], [300, 250]]} />*/}
+                            {/*    </div>*/}
+                            {/*</Card>*/}
                         </div>
                     </Col>
                     <Col md={5} className="container-fluid" style={{maxWidth: 641}}>
-                        <Row>
-                            <div className="abc">
-                                <div className="d-inline-block">
-                                    <img style={{margin: '8px 20px 20px 18px', height: '50px', borderRadius: '5px'}}
-                                         alt="" src={pic2}/>
-                                </div>
-                                <div className="d-inline-block ">
-                                    <div className="f-375">WhatsApp Direct</div>
-                                </div>
-                                <div className="d-flex justify-content-between">
-                                    {
-                                        user && user.email ?
-                                            <div style={{textAlign: "center"}}>
-                                                <Popconfirm placement="bottomLeft" title={text2}
-                                                            onConfirm={confirmLogOut}
-                                                            okText="Yes"
-                                                            cancelText="No">
-                                                    <Avatar size={50}
-                                                            icon={<img src={(user && user.photoURL)} alt=""/>}/>
-                                                </Popconfirm>
-                                                <div className="userName"> {(user && user.displayName) || ""}</div>
-                                            </div>
-                                            : <></>
-                                    }
-                                </div>
-                            </div>
-                        </Row>
-                        <Row>
-                            <div className="abc1" style={{width: '100%'}}>
-                                <div className="d-flex" style={{width: '75%'}}>
-                                    <div className="d-inline-block">
-                                        <img style={{margin: '8px 20px 20px 18px', height: '50px', borderRadius: "5px"}}
-                                             alt="" src={pic2}/>
-                                    </div>
-                                    <div className="d-inline-block r-375">
-                                        <div style={{
-                                            marginTop: '20px',
-                                            marginBottom: '15px',
-                                            color: 'white',
-                                        }}>WhatsApp
-                                            Direct
-                                        </div>
-                                    </div>
-                                </div>
-                                <div className="d-flex justify-content-between">
-                                    {
-                                        user && user.email ?
-                                            <div style={{textAlign: "center"}}>
-                                                <Popconfirm placement="bottomLeft" title={text2}
-                                                            onConfirm={confirmLogOut}
-                                                            okText="Yes"
-                                                            cancelText="No">
-                                                    <Avatar size={50}
-                                                            icon={<img src={(user && user.photoURL)} alt=""/>}/>
-                                                </Popconfirm>
-                                                <div style={{
-                                                    textAlign: 'center',
-                                                    fontSize: '12px',
-                                                    width: '100%',
-                                                    display: 'inline-table'
-                                                }}> {(user && user.displayName) || ""}</div>
-                                            </div>
-                                            : <></>
-                                    }
-
-                                </div>
-                            </div>
-                        </Row>
                         <Row style={{marginBottom: "10px"}}>
                             <Col md={4} sm={3} xs={4} lg={3} xl={3} style={{paddingRight: 0}}>
                                 <Select style={{width: '92%', textAlign: "center", background: '#707070 !important'}}
@@ -875,8 +838,6 @@
                         </div>
                         <span style={{color: "red", marginTop:-15}}>{errors.Message}</span>
 
-
-
                         <Row>
                             <Col md={6} sm={6} xs={6} lg={6} xl={6} style={{padding: "0px 5px 0px 15px"}}>
                         <span className="whatsappSend">
@@ -1232,24 +1193,22 @@
                         {/* eslint-disable-next-line react/jsx-no-undef */}
                         <Divider style={{color: "white"}}><ArrowDownOutlined/> DOWNLOAD THE
                             APP <ArrowDownOutlined/></Divider>
-                        <a href='https://play.google.com/store/apps/details?id=com.whatsapp'><img src={img1} width="45%"
-                                                                                                  height="10%" alt=""/></a>
-                        <a href='https://apps.apple.com/us/app/whats-direct-chat-and-web/id1481182077'><img src={img2}
-                                                                                                            width="45%"
-                                                                                                            height="10%"
-                                                                                                            style={{float: "right"}}
-                                                                                                            alt=""/></a>
+                        <div className="d-flex justify-content-between">
+                        <a href='https://www.microsoft.com/store/apps/9NNZMXGTVRT9' target="_blank"><img src={img1} height={70} width={70} alt=""/></a>
+                        <a href='https://play.google.com/store/apps/details?id=com.whatsapp' target="_blank"><img src={img2}  height={70} width={70} alt=""/></a>
+                        <a href='https://apps.apple.com/us/app/whats-direct-chat-and-web/id1481182077' target="_blank"><img src={img3} height={70} width={70} alt=""/></a>
+                        </div>
 
                     </Col>
                     <Col md={3}  style={{width: 320}}>
                         <div className="site-card-border-less-wrapper">
-                            {/*<Card title="Card title" >*/}
-                            {/*</Card>*/}
-                            <Card className="card" title="Card tittle" style={{width: "100%"}}>
-                                <div id="div-gpt-ad-1612775480075-0" >
-                                    <GPT adUnitPath='/419163168/https:////whatsdirectchat.com.Banner' slotSize={[[320, 50], [300, 100], [300, 250]]} />
-                                </div>
+                            <Card title="Card title" >
                             </Card>
+                            {/*<Card className="card" title="Card tittle" style={{width: "100%"}}>*/}
+                            {/*    <div id="div-gpt-ad-1612775480075-0" >*/}
+                            {/*        <GPT adUnitPath='/419163168/https:////whatsdirectchat.com.Banner' slotSize={[[320, 50], [300, 100], [300, 250]]} />*/}
+                            {/*    </div>*/}
+                            {/*</Card>*/}
                         </div>
                     </Col>
                 </Row>
